src/helm/benchmark/window_services/bloomz_window_service.py

```diff
@@ -0,0 +1,35 @@
+from .local_window_service import LocalWindowService
+from .tokenizer_service import TokenizerService
+
+
+class BloomzWindowService(LocalWindowService):
+    def __init__(self, service: TokenizerService):
+        super().__init__(service)
+
+    @property
+    def max_sequence_length(self) -> int:
+        """
+        The model was trained with a sequence length of 2,048.
+        Source: https://huggingface.co/bigscience/bloom
```

Comment:
Update the link. Also, it's probably correct, but just want to make sure the sequence length is 2048.